code/modules/mob/living/living_defense.dm

```diff
@@ -45,7 +45,7 @@
 	if(ishuman(src) && isitem(used_weapon))
 		var/mob/living/carbon/human/H = src
 		var/obj/item/I = used_weapon
-		if((H.has_organ(BP_SPCORE) || mutations.len) && (SANCTIFIED in I.aspects))
+		if((H.faction == "spiders" || mutations.len) && (SANCTIFIED in I.aspects))
```

Comment:
does is_carrion(H) not work?
my main concern is if someone made some spider equivalent of fuhrerole.

Reply:
A spiderole making you vulnerable to sanctifier sounds like a feature

Reply:
Yea, kinda new to this coding bit. A combo of not knowing that is_carrion(H) existed, and realizing that a "spiderole" probably wouldn't exist due to it being a carrion false flag, and I went to the one I figured would work. I'll try it out.

Reply:
Works like a charm. Thanks for the info on that.